modules/osdk-building-ansible-operator.adoc

```diff
@@ -208,7 +208,7 @@ Resource Definition (CRD) the Operator will be watching. Deploy the `Memcached`
 CRD:
 +
 ----
-$ oc create -f deploy/crds/cache_v1alpha1_memcached_crd.yaml
+$ oc create -f ddeploy/crds/cache.example.com_memcacheds_crd.yaml
```

Comment:
should be deploy/crds/cache.example.com_memcacheds_crd.yaml

Reply:
@emmajiafan Sorry for the typo. Fixed!